<launch>

    <arg name="model" default="lbpcascade" />

    <node pkg="cv_camera" exec="cv_camera_node" name="cv_camera_node">
        <param name="rate" value="30.0"/>
        <param name="device_id" value="0"/>

        <remap from="image_raw" to="image"/>
    </node>

    <node pkg="face_cropping" exec="peer_image_mock.py" name="peer_image_mock"/>

    <group><include file="$(find-pkg-share face_cropping)/launch/face_cropping_webrtc.launch.xml">
        <arg name="input_camera_topic" value="peer_image"/>
        <arg name="output_camera_topic" value="peer_image/cropped"/>
        <arg name="face_detection_model" value="$(var model)"/>
    </include></group>

    <node pkg="face_cropping" exec="peer_image_display.py" name="peer_image_display">
        <remap from="peer_image" to="peer_image/cropped"/>
    </node>

</launch>
